apps/wing/src/cli.ts

```diff
@@ -92,7 +92,10 @@ async function main() {
       () => {
         process.env.WING_DISABLE_ANALYTICS = "1";
       }
-    );
+    )
+    .option("--no-colors", "Disable colors for the CLI", () => {
```

Comment:
Most CLIs I've seen have this as singular rather than plural, probably to match the env and the general wording of https://no-color.org/

```suggestion
    .option("--no-color", "Disable colors for all output", () => {
```

@eladb Since you filed the issue I figured you may have a thought about this

Reply:
@MarkMcCulloh I thought the same as you, and I actually named the option with singular name before noticing the issue was asking for plural.

Reply:
@garysassano - let's go ahead and change that to `--no-color` as @MarkMcCulloh and you pointed out, no need to wait for @eladb on that one.

Reply:
Thanks for not waiting for me 🙏